english/bob-barker/barker.md

```diff
@@ -20,4 +20,6 @@ You hesitate just a moment before replying...
 
 [The red door](2050s/2050s.md)
 
-[Grab some coffee?](../coffee/coffee.md)
\ No newline at end of file
+[Grab some coffee?](../coffee/coffee.md)
+
+[Become a marshmellow](https://github.com/KellyBurdine/create-your-own-adventure/blob/master/english/becomemarshmallow/becomemarshmallow.md)
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.